test/unit/app/renderer/components/common/contextMenu/contextMenuItemTest.js

```diff
@@ -0,0 +1,168 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* global describe, before, after, it */
+
+const Immutable = require('immutable')
+const mockery = require('mockery')
+const assert = require('assert')
+require('../../../../../braveUnit')
+const {mount} = require('enzyme')
+
+describe('ContextMenuItem unit test', function () {
+  let ContextMenuItem
+
+  before(function () {
+    mockery.enable({
+      warnOnReplace: false,
+      warnOnUnregistered: false,
+      useCleanCache: true
+    })
+    ContextMenuItem = require('../../../../../../../app/renderer/components/common/contextMenu/contextMenuItem')
+    Array.prototype.contains = function (item) { return this.includes(item) } // eslint-disable-line
```

Comment:
I guess this can be removed. I don't see this being used anywhere and test pass without this line

Reply:
you are correct it's working without this one now, so I will remove it. Thank you